src/unity/lib/extensions/model_base.cpp

```diff
@@ -246,4 +241,32 @@ void model_base::register_docstring(
   m_docstring[fnname] = docstring;
 }
 
+  /**
+   * Save a toolkit class to disk.
+   *
+   * \param sidedata Any additional side information
+   * \param url The destination url to store the class.
+   */
+  void model_base::save_model_to_file(const variant_map_type& side_data,
+                          const std::string& url) {
+    std::shared_ptr<model_base> m =
+        std::dynamic_pointer_cast<model_base>(this->shared_from_this());
+
+    turi::get_unity_global_singleton()->save_model(m, side_data, url);
+  }
+
+  /**
+   * Save a toolkit class to a data stream.
+   *
+   * \param sidedata Any additional side information
+   * \param url The destination url to store the class.
```

Comment:
Documentation doesn't match function signature. Probably a copy-paste error.

Reply:
Good catch! Thank you